frontend/styles/UniOverview.module.css.d.ts

```diff
@@ -0,0 +1,4 @@
+export const overviewBody: string;
+export const uniListView: string;
+export const uniListItem: string;
+export const numOfAccepted: string;
```

Comment:
what are these for?

Reply:
ugh, the styling from the css modules was not working and I thought that this might solve it but, I just made a stupid mistake that I had quotes " in the css classes (stupid, I know, I just haven't used vanilla css for some time and it took me some time to notice it). So it's not necessary anymore, I have it removed it locally, will push the update without it to this branch.